commit e0a52c0a06c2d24139e5cc290c6af52e05b15229
Author: q66 <q66@chimera-linux.org>
Date:   Sat Apr 2 19:24:04 2022 +0200

    disable remove_test
    
    This one assumes presence of ladspa-swh-plugins.

diff --git a/examples/Makefile.am b/examples/Makefile.am
index 3960aa7..18a1396 100644
--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -1,5 +1,5 @@
 TESTS = showtaxonomy dumpstatements export_test \
-	subclass_test add_test instances_test remove_test multi_test \
+	subclass_test add_test instances_test multi_test \
 	setting_test scale_test
 
 check_PROGRAMS = showdefaults $(TESTS) remove_test
